.gitignore

```diff
@@ -3,3 +3,5 @@
 /packages/*/lib/
 /packages/*/node_modules/
 /packages/test/servers/tmp/
+
+.vscode
```

Comment:
I try to avoid putting machine/editor-specific ignore patterns in per-project ignore files. This should go in a [global gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/), or if you have other projects where you actually do want to commit `.vscode`, then you can put it in `.git/info/exclude` which is specific to your copy of this repo.